modulefiles/module_base.wcoss2.lua

```diff
@@ -3,11 +3,11 @@ Load environment to run GFS on WCOSS2
 ]])
 
 load(pathJoin("PrgEnv-intel", "8.1.0"))
-load(pathJoin("craype", "2.7.10"))
+load(pathJoin("craype", "2.7.13"))
 load(pathJoin("intel", "19.1.3.304"))
-load(pathJoin("cray-mpich", "8.1.9"))
+load(pathJoin("cray-mpich", "8.1.7"))
```

Comment:
Why is the `cray-mpich` version reverting here?

Reply:
I'm following what is here : https://github.com/ufs-community/ufs-weather-model/blob/c22aaadfb8dafb5687e2f3bf6416e8f11e3f9a1d/modulefiles/ufs_wcoss2.intel.lua 

I was having trouble getting modules to load so I checked more than just FMS and ESMF which are the usual updates.

Reply:
Were you having module load issues with `cray-mpich/8.1.9` set but those issues were resolved by going back to `cray-mpich/8.1.7`? Can you provide more background on the module load issue? Thanks!

Reply:
Unless there is an issue with that version, I am wary of stepping backwards with versions of things like cray-mpich, which NCO has mostly dictated the version of. We use `8.1.9` in GFSv16 ops (including in the ufs-weather-model branch for v16) so I'm actually surprised their develop branch hasn't updated yet.

FYI, we don't yet force module versions from the workflow level via versions files (like we do in GFSv16) but that is coming once the GSI can move to the newer intel (it's stuck on intel 2018 right now). Once we can introduce version files we will be pushing components to align versions and address any issues related to that.

For now, if there is an issue with `cray-mpich/8.1.9` I'd like to take a look and see what can be done to avoid us rolling the workflow level backwards when we have to move forward soon anyway. If you have a chance to re-create the issue or provide guidance so I can, please let me know. Thanks!

Reply:
I'll see if I can back this change out.

Reply:
Cool, sounds good! Let me know if you run into any issues and I can help. Thanks!